src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,31 @@
+package pages;
+
+import io.qameta.allure.Step;
+import org.openqa.selenium.*;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.interactions.Actions;
+
+public class HomePage extends AbstractPage {
+
+    private final By signInBtnLocator = By.linkText("Sign In");
+    private final By topsLinkLocator = By.linkText("Tops");
+    private final By womenLinkLocator = By.xpath("//nav[@class='navigation']//span[contains(text(), 'Women')]");
```

Comment:
you could use css here.
for example 'nav a[href$="/women.html"]'

Reply:
Replaced it with your variant, thank you